showcase/src/app/demo-ui/pages/date-range-picker/demo-date-range-picker-page.component.html

```diff
@@ -1,80 +1,57 @@
 <h1 class="mat-display-3" translate>SHOWCASE.DEMO.DATE_RANGE_PICKER.TITLE</h1>
 <section class="stark-section">
 	<h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
-	<example-viewer [extensions]="['HTML', 'TS']" filesPath="date-range-picker/full" exampleTitle="SHOWCASE.DEMO.DATE_RANGE_PICKER.TITLE">
-		<div>
-			<stark-date-range-picker
-				rangePickerId="id-range-picker"
-				rangePickerName="name-range-picker"
-				[isDisabled]="disabled"
-				[dateFilter]="customDateFilter"
-				[startDate]="startDate"
-				startDateLabel="STARK.DATE_RANGE_PICKER.FROM"
-				[endDate]="endDate"
-				endDateLabel="STARK.DATE_RANGE_PICKER.TO"
-				[startMinDate]="minDate"
-				[startMaxDate]="maxDate"
-				[endMinDate]="startDate || minDate"
-				[endMaxDate]="maxDate"
-				(dateRangeChanged)="onDateChanged($event)"
-			>
-			</stark-date-range-picker>
-		</div>
-		<mat-checkbox [(ngModel)]="disabled">isDisabled</mat-checkbox>
+	<example-viewer
+		[extensions]="['HTML', 'TS']"
+		filesPath="date-range-picker/model"
+		exampleTitle="SHOWCASE.DEMO.DATE_RANGE_PICKER.WITH_MODEL"
+	>
+		<stark-date-range-picker
+			[(ngModel)]="dateRangeModel"
+			(dateRangeChanged)="onDateModelChange()"
+			dateFilter="OnlyWeekdays"
+			[startMinDate]="today"
+			[endMinDate]="today"
+			[startMaxDate]="inOneMonth"
+			[endMaxDate]="inOneMonth"
```

Comment:
In this example the errors for min and max dates are not displayed:

![date-range-picker no errors](https://user-images.githubusercontent.com/5698318/56648904-69e76780-6684-11e9-9bb9-833cebcc0ac0.PNG)

Reply:
Indeed, there are no error messages defined by default for any validation. Should there be?

Reply:
Not by default... I mean, they should not be defined in Stark but in the demo page itself ;)

Reply:
Something similar to what we did in `src/app/demo-ui/pages/date-picker/demo-date-picker-page.component.html`

Reply:
I updated the demo with the messages